packages/commonwealth/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "jsx": "react",
     "sourceMap": true,
     "paths": {
+      "@hicommonwealth/*": ["../../libs/*/src/index.ts"],
```

Comment:
Can we not do a wildcard import here? 

commonweath doesn't need 'sitemaps' for example and we should only import what it needs.